bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -72,8 +72,9 @@
   private final AtomicInteger numberOfInMemBlockPermits;
   private MutableInt nextOffset = new MutableInt();
   private Future<?> future;
+  private final boolean backPressure;
 
-  public DataList(final String identifier, final int blockSize, final int numberOfCacheBlocks)
+  public DataList(final String identifier, final int blockSize, final int numberOfCacheBlocks, final boolean backPressure)
```

Comment:
backPressure->isBackPressureEnabled? Or better, as the Buffer server already supports back pressure based on the ability to spool blocks to a disk and it is always enabled, to name the flag to reflect that it is a support for the back pressure based on the slowest subscriber.